interpreter/cling/lib/MetaProcessor/MetaProcessor.cpp

```diff
@@ -395,7 +395,11 @@ namespace cling {
       }
     }
 
+#ifdef LLVM_ON_WIN32
```

Comment:
Can we merge these code paths by providing a second argument to the `ifstream` constructor?